WebPortal/Services/MetadataBrokerService.cs

```diff
@@ -142,6 +145,12 @@ public async Task<int> SaveApprovalForm(ShareWorkflow.ApprovalForm form)
             return approvalFormEntity.ApprovalFormId;
         }
 
+        private void UpdateRequiresBlanketApproval(ApprovalForm form)
+        {
+            form.Requires_Blanket_Approval_FLAG = form.Updated_On_Going_Basis_FLAG || form.Collection_Of_Datasets_FLAG || form.Approval_InSitu_FLAG || !string.IsNullOrEmpty(form.Approval_Other_TXT);
+            form.Approval_Other_FLAG = !string.IsNullOrEmpty(form.Approval_Other_TXT);
```

Comment:
Not a big deal but I think it would look a bit nicer with Approval_Other_FLAG set first, then the blanket approval would be flag || flag || flag || flag